test/functional/dataElements/xdmObject/helpers/xdmTree.js

```diff
@@ -19,63 +19,63 @@ import {
 
 const xdmTree = createTestIdSelector("xdmTree");
 
-const getNode = async title => {
-  await switchToIframe();
-  return xdmTree
-    .find(createTestIdSelectorString("xdmTreeNodeTitleDisplayName"))
-    .withText(title)
-    .parent(createTestIdSelectorString("xdmTreeNodeTitle"))
-    .nth(0);
-};
-
 export default {
-  click: async title => {
-    const node = await getNode(title);
-    await t.click(node);
-  },
-  toggleExpansion: async title => {
-    const node = await getNode(title);
-    await t.click(
-      node
-        .parent(".ant-tree-treenode")
-        .nth(0)
-        .find(".ant-tree-switcher")
-    );
-  },
-  populationIndicator: async title => {
-    const node = await getNode(title);
+  node: title => {
+    const node = xdmTree
+      .find(createTestIdSelectorString("xdmTreeNodeTitleDisplayName"))
+      .withText(title)
+      .parent(createTestIdSelectorString("xdmTreeNodeTitle"))
+      .nth(0);
     const populationIndicator = node.find(
       createTestIdSelectorString("populationAmountIndicator")
     );
+    const expansionToggle = node
+      .parent(".ant-tree-treenode")
+      .nth(0)
+      .find(".ant-tree-switcher");
     return {
-      expectFull() {
-        return t.expect(populationIndicator.hasClass("is-full")).ok();
+      click: async () => {
+        await switchToIframe();
+        await t.click(node);
+      },
+      toggleExpansion: async () => {
+        await switchToIframe();
+        await t.click(expansionToggle);
+      },
+      populationIndicator: {
+        expectFull: async () => {
+          await switchToIframe();
+          await t.expect(populationIndicator.hasClass("is-full")).ok();
+        },
+        expectPartial: async () => {
+          await switchToIframe();
+          return t.expect(populationIndicator.hasClass("is-partial")).ok();
+        },
+        expectEmpty: async () => {
+          await switchToIframe();
+          await t.expect(populationIndicator.hasClass("is-empty")).ok();
+        },
+        expectBlank: async () => {
+          await switchToIframe();
+          await t.expect(populationIndicator.exists).notOk();
+        }
+      },
+      expectIsValid: async () => {
+        await switchToIframe();
+        await t.expect(node.hasClass("is-invalid")).notOk();
       },
-      expectPartial() {
-        return t.expect(populationIndicator.hasClass("is-partial")).ok();
+      expectIsNotValid: async () => {
+        await switchToIframe();
+        await t.expect(node.hasClass("is-invalid")).ok();
       },
-      expectEmpty() {
-        return t.expect(populationIndicator.hasClass("is-empty")).ok();
+      expectExists: async () => {
+        await switchToIframe();
+        await t.expect(node.exists).ok();
       },
-      expectBlank() {
-        return t.expect(populationIndicator.exists).notOk();
+      expectNotExists: async () => {
+        await switchToIframe();
+        await t.expect(node.exists).notOk();
```

Comment:
There's some context as to why I changed the organization of this helper. I wanted to have an `expectExists` method that would tell me if the tree itself actually existed, rather than a node on the tree. It seemed appropriate, then, not put all the node-based stuff under a `node` method. I didn't actually end up adding an `expectExists` method for the tree itself, but instead added an `editor.js` helper which has an `expectExists` that tells me whether the editor exists (the editor contains the tree), which fulfilled my needs. I left these changes in the tree though because I feel it is a better structure. Unfortunately, I had to change a bunch of the usages of this helper, which makes of a larger PR diff. Sorry.